src/stores/store-urls-custom-admin.md

```diff
@@ -60,6 +60,9 @@ As a precaution, do not try to change the Admin URL yourself unless you know how
    {:.bs-callout-info}
    This method allows you to change the Admin Path, but not the Admin URL.
 
+   {:.bs-callout-info}
```

Comment:
This new callout appears to clash with the one above it. Do we need to be clear here that "Admin path" and "Admin URL" are different? This is suggesting that one is set by the other.

Reply:
This should probably be a "tip", since it lets Cloud customers know there is a different way to set this. Many things in the Admin UI are read-only for Cloud-hosted Magento. If we made a note each time, it would reduce the quality of the user guide.

Reply:
Some of this request was documented in #1098